[2024-10-20T03:01:15.350+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T02:56:07.213676+00:00 [queued]>
[2024-10-20T03:01:15.363+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_etl.load manual__2024-10-20T02:56:07.213676+00:00 [queued]>
[2024-10-20T03:01:15.364+0000] {taskinstance.py:1361} INFO - Starting attempt 2 of 2
[2024-10-20T03:01:15.386+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load> on 2024-10-20 02:56:07.213676+00:00
[2024-10-20T03:01:15.400+0000] {standard_task_runner.py:57} INFO - Started process 112 to run task
[2024-10-20T03:01:15.403+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'crypto_etl', 'load', 'manual__2024-10-20T02:56:07.213676+00:00', '--job-id', '217', '--raw', '--subdir', 'DAGS_FOLDER/crypto_etl_dag.py', '--cfg-path', '/tmp/tmpbviscyzr']
[2024-10-20T03:01:15.405+0000] {standard_task_runner.py:85} INFO - Job 217: Subtask load
[2024-10-20T03:01:15.468+0000] {task_command.py:416} INFO - Running <TaskInstance: crypto_etl.load manual__2024-10-20T02:56:07.213676+00:00 [running]> on host c03d178edf0e
[2024-10-20T03:01:15.582+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_etl' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-10-20T02:56:07.213676+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-20T02:56:07.213676+00:00'
[2024-10-20T03:01:15.648+0000] {logging_mixin.py:154} INFO - Success: Loaded 101 recall records to recalls_db.
[2024-10-20T03:01:15.648+0000] {python.py:194} INFO - Done. Returned value was: None
[2024-10-20T03:01:15.660+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=crypto_etl, task_id=load, execution_date=20241020T025607, start_date=20241020T030115, end_date=20241020T030115
[2024-10-20T03:01:15.697+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-10-20T03:01:15.716+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
